Source/AudioQueueDirector.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  AudioQueueDirector.swift
+//  SwiftAudioPlayer
+//
+//  Created by Joe Williams on 10/03/2021.
+//
+
+import Foundation
+
+class AudioQueueDirector {
```

Comment:
I would keep this class instead of using AudioClockDirector to store the closures. Both having a class or adding a list to the AudioClockDirector has the same affect but I think logically AudioClockDirector was dealing with single audio events whereas queuing is in a different space.